app/src/main/java/com/celzero/bravedns/receiver/DeviceLockService.kt

```diff
@@ -40,7 +40,7 @@ class DeviceLockService  : Service(){
     }
 
     override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
-        if(intent != null && intent.action == ACTION_CHECK_LOCK){
+        if (intent != null && intent.action == ACTION_CHECK_LOCK) {
```

Comment:
The Intent supplied to android.content.Context#startService, as given. This may be null if the service is being restarted after its process has gone away. As per the link, the intent can be null.